packages/common/common/enums/Pay.ts

```diff
@@ -0,0 +1,6 @@
+enum Pay {
+  full_prepayment = 'full_payment',
+  prepayment = 'prepayment'
+}
+
+export default Pay
```

Comment:
```suggestion
export Pay
```
Давай везде уберем, если не возражаешь.